static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -149,7 +149,7 @@ define([
             error: function () {
                 bonzo(this.adSlot).hide();
             }.bind(this)
-        }).load();
+        });
```

Comment:
:+1: because this is simpler :-) No contexts, no constructors, no prototypes — fewer language features